examples/next/visual-tests/js/demo/src/demos/basicTwoTables/basicTwoTables.js

```diff
@@ -0,0 +1,47 @@
+import Handsontable from 'handsontable';
+
+export function initializeBasicTwoTablesDemo() {
+  const root = document.getElementById('root');
+
+  const topTable = document.createElement('div');
+  topTable.id = 'tableTop';
+
+  const input1 = document.createElement('input');
+  input1.style.margin = '10px 0';
+  input1.placeholder = 'Input 1';
+  topTable.appendChild(input1);
+
+  const example1 = document.createElement('div');
+  topTable.appendChild(example1);
+
+  root.appendChild(topTable);
+
+  const bottomTable = document.createElement('div');
+  bottomTable.id = 'tableBottom';
+
+  const input2 = document.createElement('input');
+  input2.style.margin = '10px 0';
+  input2.placeholder = 'Input 2';
+  bottomTable.appendChild(input2);
+
+  const example2 = document.createElement('div');
+  bottomTable.appendChild(example2);
+
+  root.appendChild(bottomTable);
```

Comment:
This is just a comment, please don't do anything here - I'm not very happy about this imperative code that looks like JS before jQuery :) 
We should have a helper that creates those elements from `html` which is much more readable